cmd/which-fix-version/main.go

```diff
@@ -1,17 +1,96 @@
 package main
 
 import (
+	"errors"
+	"flag"
 	"fmt"
 	"os"
+	"strings"
 
 	tea "github.com/charmbracelet/bubbletea"
 	"github.com/vpofe/which-fix-version/app"
+	"github.com/vpofe/which-fix-version/git"
 )
 
-func main() {
-	app := app.NewApp()
+func NewFindCommand() *FindCommand {
+	fc := &FindCommand{
+		fs: flag.NewFlagSet("find", flag.ContinueOnError),
+	}
+
+	fc.fs.StringVar(&fc.commitHash, "commitHash", "", "the main/master/development/custom branch commit hash to find the minimal fix version")
+	fc.fs.StringVar(&fc.url, "url", "", "git repository url")
+	fc.fs.StringVar(&fc.remoteName, "remoteName", "origin", "remote name to fetch branches from")
+	fc.fs.StringVar(&fc.developmentBranchName, "developmentBranchName", "main", "name of the central development branch")
+	fc.fs.StringVar(&fc.releaseBranchPrependIdentifiers, "releaseBranchPrependIdentifiers", "release- releases/ release/", "all string characters before the release version")
+
```

Comment:
forceNoInteraction flag could be interesting

Reply:
"when not all inputs can be guessed make it jump to interactive mode"
seems to require one flag less <3